lib/index.js

```diff
@@ -450,12 +442,19 @@ class CorePlugin {
    * @param {Object} request 
    */
   recomputeComputedFields(request) {
-    // TODO: Fix return value on error...
-    // recompute fields 
     const {index, collection} = request.input.resource
     return this.recomputeCollectionComputedFields(request, index, collection)
   }
 
+  resetComputedFields() {
+    return this.computedFieldsRepository.search({}, 0, 1000)
+    .then(r => {
+      let p = []
+      r.hits.forEach((e)=> p.push(this.computedFieldsRepository.delete(e._id)))
```

Comment:
* `p` should be `const`
* unnecessary parens around `e`
* can be simplified with a map:
`const p = r.hits.map(e => this.computedFieldsRepository.delete(e._id));`